server/src/user/service.js

```diff
@@ -70,3 +79,9 @@ exports.delete = (req, res) => {
     }))
     .catch((err) => errorHandler.handle(err, res))
 }
+
+exports.activate = (req, res) => {
+  UserModel.findOneAndUpdate({ token: req.params.token }, { isActive: true, token: '' })
+    .then(result => res.status(202).send(result))
```

Comment:
Return the object structure as I did with others 
```
{
  success: true,
  message: "Activated",
  data: result
}
```
something like this... see what is needed and for catch try to use the utility Error handler.

Reply:
done